### ISSUE TITLE
Project: Inspector Prioritization Automation

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#423]
## Description
The goal of this project is to allow Right of Way Management traffic control inspectors to better prioritize their inspection workload based on priorities identified by the division and department.

**Comment by TracyLinder** [comment]
### Update

All future phases of this project are on hold. We are however continuing to work on automating the portion of work done to-date. See the epic for this portion [here](https://github.com/cityofaustin/atd-amanda/issues/119).

**Comment by TracyLinder** [comment]
10/28/19 - R. Ward asked if the output can be pushed to KNACK, instead of an excel file in SharePoint.

**Comment by amenity** [comment]
@johnclary - putting this in the backlog until we have more PM capacity.

**Comment by TracyLinder** [comment]
Note to Self: If we rely on open data for this solution, we won't be able to include franchised information as it will not be published. 
@johnclary FYI

**Comment by johnclary** [comment]
Fair point. This adds quite a bit to the level of effort. TBH when we talked about the legal concerns I thought we kinda decided that it was OK for us to publish the street segments for franchise permits 🤔. Let's talk about this during 1:1

**Comment by johnclary** [comment]
We delivered an initial, manual prototype of this solution and need to re-prioritize further development. Since we last visited this work we have gained programmatic access to AMANDA, so we are essentially unblocked.

**Comment by TracyLinder** [comment]
### Update Oct 25, 2024

ROW Management supervisors confirmed recently that their inspection group still needs a solution for prioritizing their work. I've closed all of the current issues related to this project as they represented tasks from the work we did in 2021. I do not anticipate that DTS will build on what we created in 2021; it's more likely we will build a new solution from scratch. (TBD) Next steps will be to meet with Darren U. to discuss priority, and then put it on the DTS Product Sync agenda.

**Comment by TracyLinder** [comment]
### Update Nov 1, 2023

After feedback from the Data Science team I combed the archives looking for all existing documentation; I added the resources I found to this project index description. I scheduled a meeting for 11/13/2023 with Darren, Charlie and Rebekka to discuss the following:

- Quick background overview
- Clarify the business need (use cases)
- Look at existing project documentation together
- Next steps and timeline

**Comment by TracyLinder** [comment]
### Project Update Dec 2024

We reviewed the PowerBi prototype with ROW today and it was well received. We will develop the identified improvements and move forward with publishing the dashboard with a daily refresh cadence. We will be meeting with ROW again on Jan. 28, 2024 to gather feedback and identify next steps.

**Comment by TracyLinder** [comment]
### Project Update Jan 2025

Today we released the Inspector Prioritization PowerBI Dashboard to ROW to begin using. We will meet with stakeholders on 3/20/2025 to gather feedback and work towards closing this project. 

![Image](https://github.com/user-attachments/assets/4781d073-521c-4213-bc69-7c0777b89bfa)

**Comment by amenity** [comment]
@TracyLinder - this is being executed by the Apps team, correct? If so, can we change the "Service" label to "Service: Apps"?

**Comment by TracyLinder** [comment]
### Project Update May 2025

DTS implemented ROW's requested updates to the dashboard after our product review in March. I will schedule a project closeout meeting to include documentation of the dashboard's roadmap.
